### ISSUE TITLE
Cross references in Word not treated as links by NVDA

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#14957]
### Steps to reproduce:

1. Open Microsoft Word
2. Insert a heading (control+alt+1)
3. On a separate line, create a cross-link to it: alt+s, r, f, then down arrow to heading, press enter to insert the cross link then escape to close the dialog.
4. Press NVDA+space to put NVDA in browse mode
5. Press NVDA+f7 to open the elements list.  Look for the link.

### Actual behavior:

No link is detected.  The cross reference DOES work as a link, but is not recognised as one by NVDA.

### Expected behavior:

Cross references should be treated as links.  This is a regression - it was initially included in 2016.4 with PR #6242.  I suspect it has regressed since moving to UIA for Word?

#### NVDA version:

2023.1 installed

#### Windows version:

Windows 11 (64-bit) Version: 22H2, Build: 22621.1702 

#### Name and version of other software in use when reproducing the issue:

Office 365 (64-bit) Version: 16.0.16327.20248

**Comment by XLTechie** [comment]
@Qchristensen can you check the first ("bad") table of contents in [14956example1.docx](https://github.com/nvaccess/nvda/files/14749518/14956example1.docx) from issue #14956?

In this document, at least in Word 2019, the cross references are not spoken as links (which I assume is true in your example as well, although it isn't stated), but additionally they can not be interacted with as links in browse mode, which is contrary to your results.
You say:
> The cross reference DOES work as a link, but is not recognised as one by NVDA.
You do not say how you make it work as a link, but I presume with the mouse.

**Comment by Adriani90** [comment]
Probably related to or even duplicate of #5896.

**Comment by CyrilleB79** [comment]
> Probably related to or even duplicate of [#5896](https://github.com/nvaccess/nvda/issues/5896) and [#14956](https://github.com/nvaccess/nvda/issues/14956).

No, #5896 and #14956 are with Word object model, while this one is with UIA.

@seanbudd can you add "component/UIA-ms-office"?

**Comment by Qchristensen** [comment]
Just testing again, and the behaviour is currently:

In browse mode:
- K or NVDA+f7 now finds the link as expected.... but pressing ENTER on it does not move focus (or does to somewhere in the link or neaby but not to where it should go).
- with a link called "first", pressing K, NVDA identifies it as "f BOOKMARK irst LINK"
- with the same link, moving to it with down arrow, it is read as: "LINK f BOOKMARK irst"

In focus mode:
- You can press ENTER and the link works as expected
- with the link called "first", pressing down arrow to it, NVDA identifies it as LINK f BOOKMARK irst" (the same as in Browse mode when arrowing to it).

Using:
nvda_snapshot_alpha-55600,2c434d9b.exe 
NVDA 2025.3.3


Windows 11 (64-bit) Version: 25H2, Build: 26200.8246 
 
Office 365 (64-bit) Version: 16.0.19822.20182
